Sources/Conditions/AlphabeticCondition.swift

```diff
@@ -23,31 +23,34 @@ public struct AlphabeticCondition: Condition {
     
     public var shouldAllowViolation = true
     
+    /// Whether or not to allow Unicode letters. If `false` then only ASCII letters (A-Z, a-z) are allowed.
+    public let allowsUnicode: Bool
```

Comment:
Bool is fine, I'm sure you've thought of this already that in future if the parameters increase we may require and configuration object or dictionary of parameters etc

I think it's fine for now and may never become a problem